freetypefont.mod/freetypefont.bmx

```diff
@@ -168,7 +168,7 @@ Type TFreeTypeFont Extends BRL.Font.TFont
 
 		If TStream(src)
 			Local stream:TStream = TStream(src)
-			Local data:Byte[1024]
+			Local data:Byte[1024 * 8]

```

Comment:
A quick average of TTF files from my Fonts dir, gives 87660 bytes. So maybe 90k would cover most files...